File: core/src/commonMain/kotlin/work/socialhub/kmastodon/entity/pleroma/PleromaContent.java
package work.socialhub.kmastodon.entity.pleroma;

import com.google.gson.annotations.SerializedName;

/**
 * @author uakihir0
 */
public class PleromaContent {

    @SerializedName("text/plain")
    private String textPlain;

    public String getTextPlain() {
        return textPlain;
    }

    public void setTextPlain(String textPlain) {
        this.textPlain = textPlain;
    }
}

File: core/src/commonMain/kotlin/work/socialhub/kmastodon/entity/services/pleroma/PleromaContent.kt
package work.socialhub.kmastodon.entity.services.pleroma

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable
import kotlin.js.JsExport

@JsExport
@Serializable
class PleromaContent {

    @SerialName("text/plain")
    var textPlain: String? = null
}
